.gitignore

```diff
@@ -14,8 +14,11 @@
 /integration/target/
 /integration/Cargo.lock
 
-/sdk/Cargo.lock
-/sdk/target/
+/sdks/rust/Cargo.lock
```

Comment:
This change should be mentioned in the commit message or separated out into its own commit